test-poc/framework/src/main/java/org/keycloak/test/framework/oauth/OAuthCallbackServer.java

```diff
@@ -0,0 +1,60 @@
+package org.keycloak.test.framework.oauth;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.LinkedList;
+import java.util.List;
+
+class OAuthCallbackServer {
```

Comment:
Is it intended to be package visible?

Reply:
Nopes, this is an internal class to OAuthClient, and shouldn't be called from tests